content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Uninstall Pluggable Components from KubeSphere v3.1.x"
+keywords: "Installer, uninstall, KubeSphere, Kubernetes"
+description: "Learn how to uninstall each pluggable component in KubeSphere v3.1.x."
+linkTitle: "Uninstall Pluggable Components from KubeSphere v3.1.x"
+Weight: 16500
+---
+
```

Comment:
Is it better to divide this guide into different sections and add them to [Enable Pluggable Components](https://kubesphere.io/docs/pluggable-components/)? IMO, FAQ is not a suitable place for this guide.

Reply:
Thanks for your comment. I chose to put this guide as a whole in FAQ for v3.1.x and our team will refactor the documentation structure in the next release. Is that OK?

Reply:
@Patrick-LuoYu What's your opinion?

Reply:
Moved doc to the chapter Enable Pluggable Components.